src/os/bluestore/BlueStore.cc

```diff
@@ -6909,8 +6909,11 @@ int BlueStore::_open_super_meta()
       ::decode(bluefs_extents, p);
     }
     catch (buffer::error& e) {
-      derr << __func__ << " unable to read bluefs_extents" << dendl;
-      return -EIO;
+      string kv_backend = cct->_conf->bluestore_kvbackend;
```

Comment:
> If rocksdb isn't used, kvbackend can't have the key 'bluefs_extents'.

i don't understand how `bluefs_extents` is wired with rocksdb backend. could you elaborate a little bit?